data/defaults.csv

```diff
@@ -0,0 +1,6 @@
+env_setup_commands,pip install --upgrade pip<19
```

Comment:
I don't think we want to, by default, globally change `pip`, so I'd suggest `poetry run pip install --upgrade pip<19`

Reply:
see above re: poetry (we're calling the venv pip)